### PR TITLE
REF: refactor cumulative op tests from test_analytics

### PR DESCRIPTION
**Pull request by jbrockmendel**
The removal of `ix` highlighted how difficult it is to tell how thorough tests are for some funcs/methods.  This is one of several PRs intended to clarify where the tests for X live, and ideally to make that follow a standard pattern across DataFrame/Series/Index subclasses (and EA where relevant)

### CODE REVIEW

File: pandas/tests/frame/test_cumulative.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+tests.series.test_cumulative
+"""
+
+import numpy as np
+
+from pandas import DataFrame, Series
+import pandas.util.testing as tm
+
+
+class TestDataFrameCumulativeOps:
```
Comment (WillAyd) [inline pandas/tests/frame/test_cumulative.py]: Not a huge deal but probably don't even need the classes now

**Review by WillAyd — APPROVED** [review]
I'm on board with this

File: pandas/tests/frame/test_cumulative.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/frame/test_cumulative.py]: for whatever reason i still find this style more comfortable, but agreed on the "not a huge deal"

**Comment by jreback** [comment]
thanks. i kind of like the classes as well, useful for organizing, but equally can just make a directory and a no-class structure. prob not worth the churn though.

**Pull request merged**